assets/css/index.css

```diff
@@ -2,6 +2,24 @@ body {
     font-family: "Lato",-apple-system,BlinkMacSystemFont,"Roboto","Segoe UI","Helvetica Neue","Lucida Grande",Arial,sans-serif;
 }
 
+img {
+
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 50px;
+  padding-bottom: 50px;
+
+}
```

Comment:
```suggestion
img {
  display: block;
  margin-left: auto;
  margin-right: auto;
  padding-top: 50px;
  padding-bottom: 50px;
}
```

Reply:
This commit is obsolete since I moved the img construct to main.css. I did perform this change there.